src/test/java/com/browserstack/examples/rule/WebDriverProviderRule.java

```diff
@@ -40,13 +86,15 @@ public WebDriver getWebDriver(WebDriverConfiguration webDriverConfiguration, Pla
      */
     protected void succeeded(Description description) {
         LOGGER.info("Succeeded Test :: {} WebDriver Session :: {}", description.getDisplayName(), this.driver);
+        ((JavascriptExecutor) driver).executeScript("browserstack_executor: {\"action\": \"setSessionStatus\", \"arguments\": {\"status\": \"passed\", \"reason\": \"Test Passed\"}}");
```

Comment:
Create a CONSTANT variable for this with the values substitutable.

Reply:
done